tests/test_reqparse.py

```diff
@@ -723,6 +723,39 @@ def test_strict_parsing_on_partial_hit(self):
         parser.add_argument('foo', type=int)
         self.assertRaises(exceptions.BadRequest, parser.parse_args, req, strict=True)
 
+    def test_trim_argument(self):
+        req = Request.from_values("/bubble?foo= 1 &bar=bees&n=22")
```

Comment:
The trim operation will only work for request data that is a string, basically everything except JSON, which you don't have tests for here. If you were to use JSON request data for these tests, it would fail for the [reason I gave previously.](https://github.com/flask-restful/flask-restful/pull/427/files#r28646182)